app/assets/css/main.css

```
@@ -231,3 +231,5 @@ html.light {
   -ms-overflow-style: none; /* IE and Edge */
   scrollbar-width: none; /* Firefox */
 }
+
+@import "../../../registry/ui/timeslot/utilities.css";
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)